uint16 pan_position
uint16 pan_max_speed
uint16 tilt_position
uint16 tilt_max_speed